deploy/roles/.gitignore

```diff
@@ -0,0 +1 @@
+*/
```

Comment:
All subdirectories, i.e. the roles installed here via ansible-galaxy.  (You achieved a similar goal by ignoring `roles/` one level further up.)